website/blog/2018-12-12-fast-goto-definition.md

```diff
@@ -116,21 +125,25 @@ val index = Map[Symbol, Path](
 )
 ```
 
-Our challenge is to build this index as fast as possible.
+With this index, we can lookup which file defines `scala.Some` and then parse
+`Option.scala` on-demand (like we do for Java) to find the position where
+`def isEmpty: Boolean` is defined. The challenge is to efficiently build the
+index.
```

Comment:
I like the bullet point style as in the Java section:


> With index we can:
>
> - lookup which file defines `scala.Some`
> - parse `Option.scala` on-demand (like we do for Java) to find the position where
`def isEmpty: Boolean` is defined
> 
> The challenge is to efficiently build the
index.

Reply:
Good idea, I will add bullets